crates/client/sync/Cargo.toml

```diff
@@ -2,12 +2,12 @@
 authors.workspace = true
 description = "This crate is responsible to sync data"
 edition = "2021"
-homepage = "https://github.com/KasarLabs/deoxys"
+homepage = "https://github.com/madara-alliance/madara"
 license = "MIT"
-name = "dc-sync"
+name = "mc-sync"
 publish = false
-repository = "https://github.com/KasarLabs/deoxys"
-version = "0.1.0"
+repository = "https://github.com/madara-alliance/madara"
+version = "0.7.0"
```

Comment:
why this crate does not take the attributes of the workspace

Reply:
changed it to match workspace attribute